packages/contracts/contracts/core/component/dao-authorizable/bases/DaoAuthorizableBase.sol

```diff
@@ -16,7 +16,7 @@ abstract contract DaoAuthorizableBase is Context {
 
     /// @notice Returns the DAO contract.
     /// @return IDAO The DAO contract.
-    function getDAO() external view returns (IDAO) {
+    function getDAO() public view returns (IDAO) {
```

Comment:
```suggestion
    function getDAO() external view returns (IDAO) {
```
Making this public is not necessary. External is sufficient since `IDAO internal dao` can be accessed by parents.
Another option would be to make `IDAO public dao`.